src/component/container/App/Subscription/SubscriptionContents/SubscriptionContentsContainer.tsx

```diff
@@ -159,7 +159,7 @@ export class SubscriptionContentsContainer extends BaseContainer<SubscriptionCon
                     <div
                         className="SubscriptionContentsContainer-footerPadding"
                         style={{
-                            height: 1000
+                            height: 2000
```

Comment:
Probably, use `100vh` instead of it.
`vh` is view port height that is calcurated by your view port size.

Reply:
Good idea!
Please close this PR and fix it.

Reply:
Can you commit it into this PR?